code/modules/organs/wound.dm

```diff
@@ -119,19 +120,29 @@
 		src.germ_level = max(src.germ_level, other.germ_level)
 		src.created = max(src.created, other.created)	//take the newer created time
 
+	//forces an infection, and bleeding regardless of damage or stage
+	proc/force_infect()
+		bleed_threshold = 4 	//Will always start bleeding, making the infection worse if untreated
+		forced_infected = TRUE
+		germ_level = INFECTION_LEVEL_ONE + 1
 	// checks if wound is considered open for external infections
 	// untreated cuts (and bleeding bruises) and burns are possibly infectable, chance higher if wound is bigger
 	proc/infection_check()
-		if (damage < 10)	//small cuts, tiny bruises, and moderate burns shouldn't be infectable.
-			return 0
-		if (is_treated() && damage < 25)	//anything less than a flesh wound (or equivalent) isn't infectable if treated properly
-			return 0
 		if (disinfected)
-			germ_level = 0	//reset this, just in case
-			return 0
-
+			if(germ_level > INFECTION_LEVEL_ONE)
```

Comment:
The reshuffling here may have caused external wound infections to be treatable way too easily, as all germs are wiped upon disinfection from an external wound now.